### ISSUE TITLE
Dashboard breaks after deleting plex library

### ISSUE THREAD

**Issue by SnippetSpace** [RaunoT/plex-rewind#217]
### Plex Rewind Version

3.01

### Description

After deleting a library from plex, and deleting that library and all of its statistics from tautulli, plex rewind throws the default 404 page when navigating to the dashboard.

### Steps to reproduce

1. Delete plex library
2. Delete same library in tautulli
3. delete plex rewind config file, restart container, reconfigure (old library indeed does not show up)
4. click on "dashboard" -> 404

### Platform

desktop

### Device

PC

### Operating System

Windows 11

### Browser and version

Edge 127

### Additional context

Not sure how to pull server logs. I have it running on k3s. Are there logs stored somewhere int he container file system I can retrieve for you?

**Comment by RaunoT** [comment]
What is the slug of the dashboard url? Is it trying to go to the deleted library? E.g `/dashboard/tv-shows`.
Have you tried re-saving `/settings/general`? That (or any other settings page for that matter) should revalidate the cache (including libraries info) thus updating the dashboard link on the homepage to the first existing library.

**Comment by SnippetSpace** [comment]
How strange, I fixed it? 
Initially I got no slug when clicking from the home page.
![image](https://github.com/user-attachments/assets/238d03b9-082a-48e4-9c0b-dcb4658d0b82)

However when manually adding an existing slug: https://url.com/dashboard/movies the dashboard does load:
![image](https://github.com/user-attachments/assets/95bb2e05-477a-4268-949e-3448e3164bce)

Since I manually accessed the dashboard things fixed itself and the link on the home page now redirects to https://url.com/dashboard/movies

I am not sure why or what happened. As stated, the container was fresh as was the settings file so not sure why it broke initially, might have been some kind of browser caching?

Can't test now, guess this can be closed.
